byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -49,6 +49,57 @@ \subsubsection{Delegation interface rules}
 \label{sec:delegation-interface-rules}
 
 \begin{figure}[htb]
+  \begin{equation}
+    \label{eq:rule:delegation-interface}
```

Comment:
This has the same label as the equation just above it. I suppose you copy-pasted it and forgot to update it. Just change it to something else.

Reply:
To detect this it is useful to look at the output of the compilation:

```
Latexmk: Found input bbl file 'ledger-spec.bbl'
Latexmk: Log file says output to 'ledger-spec.pdf'
Latexmk: List of undefined refs and citations:
  Label `eq:rule:delegation-interface' multiply defined
  Label `eq:rule:delegation-interface@cref' multiply defined
Latexmk: Found bibliography file(s) [references.bib]
Latexmk: Summary of warnings from last run of (pdf)latex:
  Latex found 2 multiply defined reference(s)
Latexmk: All targets (ledger-spec.pdf) are up-to-date
```

Reply:
thanks, fixed